<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutotuneShareForm</class>
 <widget class="QDialog" name="AutotuneShareForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Share Autotune Results</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QTabWidget" name="otherInfoTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="vehicleInfoTab">
      <attribute name="title">
       <string>Vehicle Information</string>
      </attribute>
      <layout class="QFormLayout" name="vehicleTabLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lblType">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="acType">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The vehicle type, e.g. QuadX. Auto-filled if a board is connected.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblBoard">
         <property name="text">
          <string>Board</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="acBoard">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The flight controller board used. Auto-filled if a board is connected.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maxLength">
          <number>64</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblWeight">
         <property name="text">
          <string>Weight (g)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="acWeight">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;All up weight of your vehicle. This should be as flown during the autotune session, including batteries, cameras etc.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="inputMask">
          <string notr="true">00000</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblVehicleSize">
         <property name="text">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="acVehicleSize">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For &lt;span style=&quot; font-weight:600;&quot;&gt;multirotors&lt;/span&gt;: the spacing between motors in millimeters, e.g. 250 mm for a 250 class.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="inputMask">
          <string notr="true">0000</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblBatteryCells">
         <property name="text">
          <string>Battery Cells</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="acBatteryCells">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The number of LiPo cells, e.g. 3S.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="currentText">
          <string>3</string>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblMotors">
         <property name="text">
          <string>Motors</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="acMotors">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The brand and model of motor.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maxLength">
          <number>64</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblEscs">
         <property name="text">
          <string>ESCs</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="acEscs">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The brand and type of speed controller.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maxLength">
          <number>64</number>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblProps">
         <property name="text">
          <string>Propellers</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="acProps">
         <property name="maxLength">
          <number>64</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="observationsTab">
      <attribute name="title">
       <string>Observations</string>
      </attribute>
      <layout class="QVBoxLayout" name="observationsTabLayout">
       <item>
        <widget class="QLabel" name="lblObservations">
         <property name="text">
          <string>Observations</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="teObservations">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Any observations you made during the tune (e.g. weather conditions if windy), and about the performance of the resulting tune.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="btnClipboard">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Copy the results to your clipboard in plain text format.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Copy to Clipboard</string>
     </property>
     <property name="icon">
      <iconset resource="configgadget.qrc">
       <normaloff>:/configgadget/images/ic_content_paste_48px.svg</normaloff>:/configgadget/images/ic_content_paste_48px.svg</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCommandLinkButton" name="btnDatabase">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Send the results to the dRonin autotune results database. This is used to assess trends. A unique device ID is sent with this data.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Send to Database</string>
     </property>
     <property name="icon">
      <iconset resource="configgadget.qrc">
       <normaloff>:/configgadget/images/ic_cloud_upload_48px.svg</normaloff>:/configgadget/images/ic_cloud_upload_48px.svg</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress">
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>45</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="btnBar">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>acType</tabstop>
  <tabstop>acBoard</tabstop>
  <tabstop>acWeight</tabstop>
  <tabstop>acVehicleSize</tabstop>
  <tabstop>acBatteryCells</tabstop>
  <tabstop>acMotors</tabstop>
  <tabstop>acEscs</tabstop>
  <tabstop>teObservations</tabstop>
  <tabstop>btnClipboard</tabstop>
  <tabstop>btnDatabase</tabstop>
  <tabstop>otherInfoTab</tabstop>
 </tabstops>
 <resources>
  <include location="configgadget.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnBar</sender>
   <signal>rejected()</signal>
   <receiver>AutotuneShareForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
